spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin2/ZipkinProperties.java

```diff
@@ -84,6 +91,14 @@ public void setBaseUrl(String baseUrl) {
 		this.baseUrl = baseUrl;
 	}
 
+	public String getApiPath() {
+		return apiPath;
```

Comment:
`return this.apiPath` - let's be consistent

Reply:
will include in the next push. Thanks @marcingrzejszczak